### PR TITLE
Allow user to edit their own email address

### PR DESCRIPTION
**Pull request by brianlovin**
**Status**
- [ ] WIP
- [x] Ready for review
- [ ] Needs testing

**Deploy after merge (delete what needn't be deployed)**
- api
- hyperion (frontend)

Note: changing email address will send a validation email which the user has to click to actually update their email in the db. This is to prevent spoofing.

### CODE REVIEW

File: api/mutations/user/editUser.js
Comment (mxstbr) [inline api/mutations/user/editUser.js]: Do we want to expose that information?

File: api/mutations/user/editUser.js
Comment (mxstbr) [inline api/mutations/user/editUser.js]: Which duplicate check?

Comment (brianlovin) [reply api/mutations/user/editUser.js]: Whoops, was able to remove it. Will remove this comment.

File: api/mutations/user/editUser.js
Comment (brianlovin) [reply api/mutations/user/editUser.js]: Hmm...good point.

Comment (brianlovin) [reply api/mutations/user/editUser.js]: We'll have to just keep an eye out if people bump into this without being malicious - in the meantime I'll obfuscate the error message so that it can't be determined.

**Comment by brianlovin** [comment]
Ping @mxstbr for final review.

**Review by mxstbr — APPROVED** [review]
Oh sorry I didn't realize you were waiting on a review for that last commit! LGTM let's ship it

**Pull request merged**
